lib/oauth/request_proxy/action_controller_request.rb

```diff
@@ -67,7 +67,7 @@ def parameters_for_signature
 
       params.
         join('&').split('&').
-        reject(&:blank?).
+        reject { |s| s.match(/\A\s*\z/) }.
```

Comment:
hi! the reason for the regexp is not `nil` (`nil.match` is an error but as you say there's no chance for `nil`) but non-empty strings consisting of only whitespace. Those I don't now if can be ruled out, that's why I preferred to keep the semantics (but if they could be ruled out, of course `empty?` would be nicer :smile:)